pkg/pillar/hypervisor/kvm_test.go

```diff
@@ -2847,11 +2858,20 @@ func TestPCIAssignmentsTemplateFillMultifunctionDevice(t *testing.T) {
 	}
 
 	wr := bytes.Buffer{}
+	multifunctionDevices := multifunctionDevGroup(pciAssignments)
+	addrAllocator := pciAddressAllocator{
+		pciAssignments:       pciAssignments,
+		multifunctionDevices: multifunctionDevices,
+	}
+	if err := addrAllocator.allocate(); err != nil {
+		t.Error(err)
+	}
+
 	p := pciAssignmentsTemplateFiller{
-		multifunctionDevices: multifunctionDevGroup(pciAssignments),
+		multifunctionDevices: multifunctionDevices,
 		file:                 &wr,
 	}
-	p.do(&wr, pciAssignments, 0)
+	p.do(&wr, pciAssignments)
```

Comment:
why are we removing 0 here from parameters?

Reply:
PCI addresses are now decided by pciAddressAllocator, not pciAssignmentsTemplateFiller.